.changeset/swift-ladybugs-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/sidebar-settings": patch
+---
+
+add highlight border settings for sidebar
```

Comment:
for the message use the same format than for the PR name (fix/feat/...) and note that this package is agnostic, this is related to the fontInput, not the sidebar:
```suggestion
feat: add highlight border settings for fontInput block
```